python_course_1604/class_02/exercize_04_iterators.py

```diff
@@ -21,12 +21,22 @@ def execute_commands(commands):
 
     iterator = iter(commands)
 
+    skip_flag = False
+
     while True:
         # HINT: look how next function behave when sequence terminates
 
-        command = next(iterator)
-
-        if command == 'ping':
+        try:
+            command = next(iterator)
+        except StopIteration:
+            break
+
+        if skip_flag:
```

Comment:
This exercise didn't requires to implement skip command. This is way there is no tests for that case and you wasn't able to have a 100% of coverage. There is an error in specs that requires to use continue statement too but it is a mistake. My apologies for that.